cli/cmd/create.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/configure"
+	"github.com/tarantool/tt/cli/create"
+	"github.com/tarantool/tt/cli/modules"
+)
+
+var (
+	appName            string
+	forceMode          bool
+	nonInteractiveMode bool
+	varsFromCli        *[]string
+	varsFile           string
+)
+
+// NewCreateCmd creates an application from a template.
+func NewCreateCmd() *cobra.Command {
+	var createCmd = &cobra.Command{
+		Use:   "create [TEMPLATE] [flags]",
+		Short: "Creates an application.",
```

Comment:
The dot at the end of a sentence in the message is at odds with the general style of the rest help of the commands.

Reply:
Fixed.